assets/product-info.js

```diff
@@ -8,6 +8,7 @@ if (!customElements.get('product-info')) {
       cartUpdateUnsubscriber = undefined;
       swapProductUtility = undefined;
       abortController = undefined;
+      inflightRequestProductUrl = null;
```

Comment:
This variable is used to track which product url should be requested when a buyer is clicking between options before the section renderer request resolves.

For example, we have a combined listing Pants with parent option color and child option size.

If the buyer is on `black/small` and clicks `orange` and then `medium` before the `orange` request resolves, the final request should be for `orange/medium`.